cfme/tests/automate/custom_button/__init__.py

```diff
@@ -27,3 +30,17 @@ def log_request_check(appliance, expected_count):
         expected_count: expected request count in automation log
     """
     return check_log_requests_count(appliance=appliance) == expected_count
+
+
+class TextInputDialogView(View):
+    """ This is view comes on different custom button objects for dialog execution"""
+
+    title = Text("#explorer_title_text")
+    service_name = TextInput(id="service_name")
+    submit = Button("Submit")
+    cancel = Button("Cancel")
+
+    @property
+    def is_displayed(self):
+        # This is only for wait for view
```

Comment:
This is fine like it is for now, but it sounds like you need to make a decision on either raising `NotImplementedError` here via `cfme.exceptions.displayed_not_implemented` property, or add some more specific checks against the title as well.